b6func.py

```diff
@@ -301,9 +302,11 @@ def edi_resample(src, w, h, edi=None, kernel='spline16', a1=None, a2=None,
                             etype=None, pscrn=None, device=None)
 
 
-def scaled_grain(clip: vs.VideoNode, grain_w: int, grain_h: int, var: float,
+def scaled_grain(clip: vs.VideoNode, var: float = 1,
```

Comment:
`var=1` seems like a pretty strong default. I typically start with something like 0.25 and tweak from there so I'm inclined to make it that. Open to discussion on it though.